scripts/add-cluster.sh

```diff
@@ -13,25 +14,126 @@ user_help () {
 
 login_to_cluster() {
     if [[ ${SINGLE_CLUSTER} != "true" ]]; then
-        if [[ -z "${IS_MINISHIFT}" ]]; then
-          if [[ -z ${KUBECONFIG} ]]; then
-            echo "Write the server url for the cluster type: $1:"
-            read -p '> ' CLUSTER_URL
-            echo "Provide the token to log in to the cluster ${CLUSTER_URL}:"
-            read -sp '> ' CLUSTER_TOKEN
-            echo ""
-            oc login --token=${CLUSTER_TOKEN} --server=${CLUSTER_URL}
-          else
-            oc config use-context "$1-admin"
-          fi
-        else
-            echo "Switching to profile $1"
-            minishift profile set $1
-            oc login -u=system:admin
-        fi
+      if [[ -z ${KUBECONFIG} ]]; then
+        echo "Please specify the path to kube config file using the parameter --kube-config"
+      else
+        oc config use-context "$1-admin"
+      fi
     fi
 }
 
+create_service_account() {
+cat <<EOF | oc apply -f -
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ${SA_NAME}
+  namespace: ${OPERATOR_NS}
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ${SA_NAME}
+  namespace: ${OPERATOR_NS}
+rules:
+- apiGroups:
+  - toolchain.dev.openshift.com
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  verbs:
+  - "get"
+  - "list"
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ${SA_NAME}
+rules:
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  verbs:
+  - "get"
+  - "list"
```

Comment:
@alexeykazakov yes, I know that this won't probably work in OSD clusters, but I have a solution for that. 
We need this permission in host operator for getting the console & Che URL of the member cluster:  https://github.com/codeready-toolchain/host-operator/blob/master/pkg/controller/masteruserrecord/sync.go#L144-L193
My plan is to drop this code from the host operator, move it to member to `MemberStatus` controller and store the URLs in the `MemberStatus` CR.
I believe that this will be also a good isolation - the `ToolchainCluster` tokens will be able to modify only the `toolchain.*` resources and the information about the other resources should be passed as part of the `MemberStatus` CR.

Reply:
Yeah... I don't like the current way of obtaining/setting the console and Che URLs anyway... it currently looks more like a hack.
I'm OK to move it to the member status. It sounds reasonable. And I think we should then add it to the toolchain status too, so our reg service can obtain it from there instead of relying on individual URLs per user. Those URLs are global anyway. And we don't want to deal with updating all the users if those URLs are changed.
But for now let's do the minimum. Just enough to unblock the toolchaincluster thing.